Add Community Contributors: 2021 newspost (#7577)
ppy/osu-wiki: news/2022-06-30-community-contributors-2021.md

Some people take it upon themselves to go simply above and beyond all expectations, and it is upon their labours that the foundation of osu! has been laid all this time.

![Community Contributor badge](https://assets.ppy.sh/profile-badges/contributor.jpg)

We are privileged to recognise more of these incredible people (in no particular order):

## Contributors

[![](/wiki/shared/news/2022-06-30-community-contributors-2021/kasumi-sama.jpg)](https://osu.ppy.sh/users/6177263)

Single-handedly leading the charge forward to elevate the osu!taiko community tournament scene, Kasumi-sama has created a towering legacy under the Global Taiko Showdown (GTS) series. Renowned for their excellent production quality, competitive spirit and considerable reach into entry-level tournaments as a whole, GTS has rapidly established itself as one of the most robust and engaged community projects seen in recent years.

> Paired with a keen artistic drive and the wherewithal to engage with many of our Featured Artists and mappers alike to produce new and original content for the community, it is safe to say that Kasumi-sama represents the latest generation of a new era of osu! creators wading knee deep into osu!'s most prominent ethic: making things more awesome.

Reviewer: ```suggestion
Paired with a keen artistic drive and the wherewithal to engage with many of our Featured Artists and mappers alike to produce new and original content for the community, it is safe to say that Kasumi-sama represents the latest generation of a new era of osu! creators wading knee-deep into osu!'s most prominent ethic: making things more awesome.
```

> Paired with a keen artistic drive and the wherewithal to engage with many of our Featured Artists and mappers alike to produce new and original content for the community, it is safe to say that Kasumi-sama represents the latest generation of a new era of osu! creators wading knee deep into osu!'s most prominent ethic: making things more awesome.

Reviewer 2: he's also an osu!catch loved captain now, could be used to show breadth of involvement or something. I'm not sure if it's important enough to mention though.

Author: the context of his broader achievements (aka GTS et al) cover enough ground in his blurb that i don't think PL involvement needs to be added on as well, esp since it's recent iirc

[![](/wiki/shared/news/2022-06-30-community-contributors-2021/frenzibyte.jpg)](https://osu.ppy.sh/users/14210502)

> A regular contributor to ongoing development efforts in osu!(lazer), frenzibyte has made himself well-known in the scene within recent years. His body of work is vast and difficult to accurately explain in a mere few paragraphs, best described instead as making efforts like the iOS version of the game possible once again for thousands of people to enjoy.

Reviewer: ```suggestion
A regular contributor to ongoing development efforts in osu!(lazer), frenzibyte has made himself well known in the scene within recent years. His body of work is vast and difficult to accurately explain in a mere few paragraphs, best described instead as making efforts like the iOS version of the game possible once again for thousands of people to enjoy.
```

> A regular contributor to ongoing development efforts in osu!(lazer), frenzibyte has made himself well-known in the scene within recent years. His body of work is vast and difficult to accurately explain in a mere few paragraphs, best described instead as making efforts like the iOS version of the game possible once again for thousands of people to enjoy.

Reviewer 2: I'm not familiar enough to add anything here myself but this feels a little short. if lazer on iOS is his main thing, it should be written to sound like more of a significant undertaking, and if not, I think you should include more examples of the work he does beyond that

Author: the development blurbs are tricky to write because an enormous amount of work can often be summed up in a single sentence at times, even if it encompasses hundreds of hours

will chew on it but i'm wary of padding it out for the sake of longer paragraphs

[![](/wiki/shared/news/2022-06-30-community-contributors-2021/susko3.jpg)](https://osu.ppy.sh/users/18945305)

> Tirelessly striving to fix bugs and bridge the differences between computing platforms, Susko3’s efforts in lazer development have proven instrumental in pushing the project forward in recent times. Their expertise in both lazer’s underlying development framework and other libraries (such as SDL) has proven integral to ongoing projects, and we are tremendously thankful for their dedication and support.

Reviewer: ```suggestion
Tirelessly striving to fix bugs and bridge the differences between computing platforms, Susko3's efforts in lazer development have proven instrumental in pushing the project forward in recent times. Their expertise in both lazer's underlying development framework and other libraries (such as SDL) has proven integral to ongoing projects, and we are tremendously thankful for their dedication and support.
```

> Tirelessly striving to fix bugs and bridge the differences between computing platforms, Susko3’s efforts in lazer development have proven instrumental in pushing the project forward in recent times. Their expertise in both lazer’s underlying development framework and other libraries (such as SDL) has proven integral to ongoing projects, and we are tremendously thankful for their dedication and support.

Reviewer 2: this one says "lazer" while elsewhere there is "osu!(lazer)", no big deal but seems random. "lazer" sounds better to me for whole article but ik peppy has had a stronger opinion for the opposite in the past...

Reviewer: i have no opinion on the current wording

Author: osu!(lazer) is apparently most correct so will change

[![](/wiki/shared/news/2022-06-30-community-contributors-2021/stanr.jpg)](https://osu.ppy.sh/users/7217455)

> A mainstay presence in the performance points development community, StanR has been at the forefront of making meaningful changes to osu!'s key performance algorithms for a considerable length of time at this point, contributing new changes of his own and vetting those made by others. While this may not sound like much at first glance, his contributions represent hundreds of hours of work and have manifested in scoring becoming fairer and more fun for everyone.

Reviewer 2: just want to double-check why "While this may not sound like much at first glance" was necessary to say, because to me it kind of does sound like a large effort. is this representing an opinion ppl would actually have?

Author: good call, will remove

[![](/wiki/shared/news/2022-06-30-community-contributors-2021/mbmasher.jpg)](https://osu.ppy.sh/users/4498616)

Taking it upon himself to bring light into the (quite literal) darkness, MBmasher turned his attention upon the scoring particulars of the Flashlight mod and found them terribly lacking. So, he boldly charted ground none before him had dared to tread, and wrote an almost entirely new algorithm from the bottom up. Before him, Flashlight was a funny novelty mod with a fairly static score multiplier, and after him, it is appropriately acknowledged in code as the nuanced and difficult playstyle that it truly is.

[![](/wiki/shared/news/2022-06-30-community-contributors-2021/walavouchey.jpg)](https://osu.ppy.sh/users/5773079)

> Ever present in the osu-wiki scene, Walavouchey is a relatively recent face who all but skyrocketed into prestige with his fastidious attention to detail as an editor and writer both. The path of a wikifriend is a long and arduous one, fraught with sprawling pull requests and sometimes unimaginably arduous tasks involving rewriting, reformatting, and restructuring lengthy articles and important explanations, and it is in all of this that Walavouchey has simply excelled without question.

Reviewer: ```suggestion
Ever present in the osu! wiki scene, Walavouchey is a relatively recent face who all but skyrocketed into prestige with his fastidious attention to detail as an editor and writer both. The path of a wikifriend is a long and arduous one, fraught with sprawling pull requests and sometimes unimaginably arduous tasks involving rewriting, reformatting, and restructuring lengthy articles and important explanations, and it is in all of this that Walavouchey has simply excelled without question.
```

> Ever present in the osu-wiki scene, Walavouchey is a relatively recent face who all but skyrocketed into prestige with his fastidious attention to detail as an editor and writer both. The path of a wikifriend is a long and arduous one, fraught with sprawling pull requests and sometimes unimaginably arduous tasks involving rewriting, reformatting, and restructuring lengthy articles and important explanations, and it is in all of this that Walavouchey has simply excelled without question.

Reviewer 2: "osu-wiki" and (later) "osu!wiki", similar to my comment about lazer wording. I think all should be changed to "osu! wiki" as that's the proper:tm: name used on the wiki itself

> Ever present in the osu-wiki scene, Walavouchey is a relatively recent face who all but skyrocketed into prestige with his fastidious attention to detail as an editor and writer both. The path of a wikifriend is a long and arduous one, fraught with sprawling pull requests and sometimes unimaginably arduous tasks involving rewriting, reformatting, and restructuring lengthy articles and important explanations, and it is in all of this that Walavouchey has simply excelled without question.

Reviewer 2: maybe add something about how he reviews basically every news post too cuz that's sort of a brand at this point.

[![](/wiki/shared/news/2022-06-30-community-contributors-2021/niva.jpg)](https://osu.ppy.sh/users/197805)

> With a storied history of contribution to osu! via the Mappers' Guild as a fourth level (maximum) member and an Mappers' Choice Awards 2020 recipient, it wouldn't be unusual to think that only the mapping scene receives the bulk of Niva's attention between his Global Moderator responsibilities and everything else.

Reviewer: ```suggestion
With a storied history of contribution to osu! via the Mappers' Guild as a fourth-level (maximum) member and a Mappers' Choice Awards 2020 recipient, it wouldn't be unusual to think that only the mapping scene receives the bulk of Niva's attention between his Global Moderator responsibilities and everything else.
```

> It turns out that contentment with that alone was not on the cards, and so Niva delved deep into the arcane arts of maintaining the osu!wiki, rising today to claim his true place as a recognized wikifriend. His efforts on this front are chiefly centred around - you guessed it - mapping and tournament related articles, making hundreds of changes to promote various other community projects and keep things informative for everyone.

Reviewer: https://osu.ppy.sh/wiki/en/Article_styling_criteria/Writing#osu!

```suggestion
It turns out that contentment with that alone was not on the cards, and so Niva delved deep into the arcane arts of maintaining the osu! wiki, rising today to claim his true place as a recognised wikifriend. His efforts on this front are chiefly centred around ⁠— you guessed it ⁠— mapping- and tournament-related articles, making hundreds of changes to promote various other community projects and keeping things informative for everyone.
```

[![](/wiki/shared/news/2022-06-30-community-contributors-2021/technocoder.jpg)](https://osu.ppy.sh/users/10338558)

> The pillar upon which the members of the osu! community who wish to play the current build of osu! (stable) on their Tim Cook-approved Apple computers, Technocoder has well and truly lived up to his name over the years, creating a litany of Wineskins and other assorted tools designed to help get macOS users situated neatly into the game.

Reviewer: ```suggestion
The pillar upon which the members of the osu! community who wish to play the current build of osu! (stable) on their Tim Cook–approved Apple computers, Technocoder has well and truly lived up to his name over the years, creating a litany of Wineskins and other assorted tools designed to help get macOS users situated neatly into the game.
```

As if this were not enough on its own, he is never far from hand in any macOS-related threads, ever ready and willing to help those who need it. His efforts have easily influenced thousands at this point, and we are enormously grateful for them.

[![](/wiki/shared/news/2022-06-30-community-contributors-2021/huu.jpg)](https://osu.ppy.sh/users/6044237)

> Entrusted with the wardenship of Project Loved by its previous maintainers, huu was saddled with no small task, and has simply excelled admirably in his time heading the project. From organizing mappers, writers, and captains alike, huu has delved right into the thick of things and helped ensure that the legacy of Project Loved was set to continue in the present day, and it is a sure thing to say that it would not be anywhere near as successful as it presently is without his dedication and attention.

Reviewer: ```suggestion
Entrusted with the wardenship of Project Loved by its previous maintainers, huu was saddled with no small task, and has simply excelled admirably in his time heading the project. From organising mappers, writers, and captains alike, huu has delved right into the thick of things and helped ensure that the legacy of Project Loved was set to continue in the present day, and it is a sure thing to say that it would not be anywhere near as successful as it presently is without his dedication and attention.
```

Though he has since left the project to move on to other interests elsewhere, we are deeply grateful for his contributions, and are proud to welcome him to the list.

[![](/wiki/shared/news/2022-06-30-community-contributors-2021/olibomby.jpg)](https://osu.ppy.sh/users/6573093)

> The mastermind behind the deceptively simple-sounding toolkit known only as Mapping Tools, OliBomby's developmental efforts have trivialized previously impossibly arduous tasks of creative complexity in the mapping scene, making things that were simply not possible before into reality. With nearly two dozen tools crammed into one package, the lifetime hours-saving of his package must number well into the thousands at this point.

Reviewer: ```suggestion
The mastermind behind the deceptively simple-sounding toolkit known only as Mapping Tools, OliBomby's developmental efforts have trivialised previously impossibly arduous tasks of creative complexity in the mapping scene, making things that were simply not possible before into reality. With nearly two dozen tools crammed into one package, the lifetime hour savings of his package must number well into the thousands at this point.
```

or maybe

```suggestion
The mastermind behind the deceptively simple-sounding toolkit known only as Mapping Tools, OliBomby's developmental efforts have trivialised previously impossibly arduous tasks of creative complexity in the mapping scene, making things that were simply not possible before into reality. With nearly two dozen tools crammed into one package, the lifetime hours saved by his package must number well into the thousands at this point.
```

It should come as no surprise that when the modding and mapping community was asked to submit nominations for our list, OliBomby received the highest number of any individual we have ever seen thus far... several times over.

[![](/wiki/shared/news/2022-06-30-community-contributors-2021/frukoyurdakul.jpg)](https://osu.ppy.sh/users/7612550)

Continuing a long-standing trend of developers striving to make the world better one quality-of-life utility at a time, frukoyurdakul has earned the eternal love of the osu! taiko community with his timidly-named utility titled "Manage Beatmap", more commonly referred to as the "SV tool" or the "fruko tool" for good reason. 

It is said that this singular utility is responsible for the birth of an entirely new mapping style, an accolade highly supported by the absolute deluge of nominations he received from the mapping and modding community.

[![](/wiki/shared/news/2022-06-30-community-contributors-2021/leoflt.jpg)](https://osu.ppy.sh/users/3668779)

> A constant presence in the tournament scene for many years and recently brought aboard as part of the official osu! World Cup management crew, LeoFLT's tireless efforts largely speak for themselves. From handling programming and statistics-related tasks, to spreadsheets, website designs and much more, his proven technical prowess is only the tip of the iceberg.

Reviewer: ```suggestion
A constant presence in the tournament scene for many years and recently brought aboard as part of the official osu! World Cup management crew, LeoFLT's tireless efforts largely speak for themselves. From handling programming- and statistics-related tasks, to spreadsheets, website designs and much more, his proven technical prowess is only the tip of the iceberg.
```

Reviewer: note that [this suggestion](https://github.com/ppy/osu-wiki/pull/7577#discussion_r910866463) has the same change

Author: unless i'm blind there's no terminating dash there nor does it join to another word, is the suggestion missing something?

[![](/wiki/shared/news/2022-06-30-community-contributors-2021/chillierpear.jpg)](https://osu.ppy.sh/users/9501251)

> A forefront designer, caster and organizer in the osu! community tournament scene long before he became a part of the official World Cup management crew in recent months, ChillierPear's contributions in the field span many different domains. His graphical work is widely believed to have encouraged a significant uptick in the overall quality expectations for many community tournaments, plus his dedication to casting and streaming have left him with a well-earned reputation amongst those who make osu!'s competitive scene tick.

Reviewer: ```suggestion
A forefront designer, caster and organiser in the osu! community tournament scene long before he became a part of the official World Cup management crew in recent months, ChillierPear's contributions in the field span many different domains. His graphical work is widely believed to have encouraged a significant uptick in the overall quality expectations for many community tournaments, plus his dedication to casting and streaming have left him with a well-earned reputation amongst those who make osu!'s competitive scene tick.
```

[![](/wiki/shared/news/2022-06-30-community-contributors-2021/cavoeboy.jpg)](https://osu.ppy.sh/users/7361815)

> The host and organizer for the largest osu! convention/LAN/gathering in the world, cavoeboy’s contributions have been integral towards building a sense of worldwide community in osu!. Attended by creator, developer, mapper, and player alike, cavoe's osu! event (COE) represents a monumental amount of effort and work that often goes chronically underappreciated, with professional firms on massively larger budgets struggling to organize events to the quality and expectations that COE reaches on the regular.

Reviewer: ```suggestion
The host and organiser for the largest osu! convention/LAN/gathering in the world, cavoeboy’s contributions have been integral towards building a sense of worldwide community in osu!. Attended by creator, developer, mapper, and player alike, cavoe's osu! event (COE) represents a monumental amount of effort and work that often goes chronically underappreciated, with professional firms on massively larger budgets struggling to organise events to the quality and expectations that COE reaches on the regular.
```

Not content with only this, cavoeboy has been involved in an entire swathe of major osu! tournaments such as Corsace (both Corsace Open and Corsace Closed) and Organized Test Tournament, making it very clear that being a cornerstone for awesome experiences in the community is more or less his way of life.

---

Times remain trying around much of the world in these past few years, which only makes us all the more grateful for those who continue to carry the osu! community upon their shoulders. We continue to grow in both size and scope as a community, and with more major developments flickering into life as the months pass, it is almost certain that we will soon be approaching a new era in osu!'s 15-year history.

To everyone who helps make the osu! community more awesome, we thank you from the bottom of our hearts, and we look forward to sharing and honouring your accolades someday soon.

> —the osu!team

Reviewer: ![image](https://user-images.githubusercontent.com/36758269/176658457-58b7ad59-1451-4c7f-9610-d1885879d920.png)

```suggestion
—the osu! team
```

Reviewer 2: to be fair,
![image](https://user-images.githubusercontent.com/9954349/176661308-815a0090-5281-43cf-8378-09cd3c4198da.png)

Reviewer: well either this or that changes i suppose
